diff --git a/appteka/__init__.py b/appteka/__init__.py
index 0ec267b..ae426c3 100644
--- a/appteka/__init__.py
+++ b/appteka/__init__.py
@@ -17,4 +17,4 @@
 
 """Initial script."""
 
-__version__ = '0.10.0'
+__version__ = '0.11.0'
diff --git a/appteka/io.py b/appteka/io.py
index 6110038..357fedf 100644
--- a/appteka/io.py
+++ b/appteka/io.py
@@ -18,7 +18,6 @@
 """Writing data to IO text buffer using queue and working in separate
 thread."""
 
-
 from collections import deque
 from threading import Thread, Lock, Event
 from time import sleep
@@ -84,12 +83,15 @@ class PyRecordThread(Thread):
         self._can_quit = Event()
         self._have_data = Event()
         self.save_func = save_func
+
         if write_on == 'time':
             self._loop_func = self._sleep_loop
             self._loop_func_args = (write_every, )
+
         elif write_on == 'data':
             self._loop_func = self._event_loop
             self._loop_func_args = ()
+
         else:
             raise RuntimeError("write_on must be 'time' or 'data'.")
 
@@ -122,36 +124,3 @@ class PyRecordThread(Thread):
             self._must_write = False
         self._can_quit.wait()
         self.join()
-
-
-def _example():
-    from random import randint
-
-    def convert_func(sample):
-        """Convert function."""
-        return '{}: {}\n'.format(*sample)
-
-    writer = QueuedWriter(write_on='time')
-    writer.set_convert_func(convert_func)
-    buf = open('output.txt', 'w')
-    writer.set_buff(buf)
-    writer.start_thread()
-
-    i = 0
-    while True:
-        try:
-            sample = (i, randint(0, 100))
-            print(sample)
-            writer.add_data(sample)
-            sleep(0.02)
-            i += 1
-        except KeyboardInterrupt:
-            print("Good bye...")
-            break
-
-    writer.stop_thread()
-    buf.close()
-
-
-if __name__ == "__main__":
-    _example()
diff --git a/release-checklist.md b/release-checklist.md
index 8b41baa..bbdf06a 100644
--- a/release-checklist.md
+++ b/release-checklist.md
@@ -1,16 +1,17 @@
 # Release checklist
 
+## Tests
+
+* All passed (`make check`)
+* Coverage >= 51% (`htmlcov`)
+
 ## Code
 
-* No `TODO` in the source code
-* All unit tests passed (`make check`)
-* The coverage by tests is not less than 40%
+* No `TODO`
 * No flakes (`make flake`)
   - Note: unused imports are allowed in deprecated modules
-* The result of pylint is not less than 9.3 (`make lint`)
-* No errors from pylint (`make lint-e`)
-* Actual version in __init__.py
-* All dependencies are relevant in setup.py
+* No lint errors (`make lint-e`)
+* Lint result >= 9.35 (`make lint`)
 
 ## Docs
 
@@ -18,6 +19,8 @@
 * All necessary modules are included to docs
 * History is updated in docs
 
-## Installation
+## Distr
 
+* Actual version  (`__init__.py`)
+* Dependencies are relevant (`setup.py`)
 * Installation successful (`python3 setup.py install --user`)
diff --git a/test/test_editor.py b/test/test_editor.py
index e8d5c5b..e1bb21d 100644
--- a/test/test_editor.py
+++ b/test/test_editor.py
@@ -1,8 +1,4 @@
-import sys
-import os
 import unittest
-
-sys.path.insert(0, os.path.abspath("."))
 from appteka.pyqt import testing
 from appteka.pyqt.code_text_edit import CodeTextEdit
 
diff --git a/test/test_multiwaveform.py b/test/test_multiwaveform.py
index b2531f7..bf0c9ba 100644
--- a/test/test_multiwaveform.py
+++ b/test/test_multiwaveform.py
@@ -1,8 +1,4 @@
-import sys
-import os
 import unittest
-
-sys.path.insert(0, os.path.abspath("."))
 from appteka.pyqt import testing
 from appteka.pyqtgraph.waveform import MultiWaveform
 
diff --git a/test/test_phasor.py b/test/test_phasor.py
index 85ca396..6c79c7f 100644
--- a/test/test_phasor.py
+++ b/test/test_phasor.py
@@ -1,10 +1,5 @@
-import os
-import sys
 import unittest
-
 from PyQt5 import QtCore
-
-sys.path.insert(0, os.path.abspath("."))
 from appteka.pyqt import testing
 from appteka.pyqtgraph import phasor
 
diff --git a/test/test_waveform.py b/test/test_waveform.py
index d63385f..9c0c20e 100644
--- a/test/test_waveform.py
+++ b/test/test_waveform.py
@@ -1,9 +1,5 @@
-import sys
-import os
 import unittest
 from PyQt5 import QtCore
-
-sys.path.insert(0, os.path.abspath("."))
 from appteka.pyqt import testing
 from appteka.pyqtgraph.waveform import Waveform
 
diff --git a/todo.org b/todo.org
index eb6ac76..645e26c 100644
--- a/todo.org
+++ b/todo.org
@@ -152,7 +152,16 @@
 ** DONE fix: waveform, phasor: update_data()
 ** DONE docs: update (renaming modules)
 ** DONE docs: update history
+* 0.11.0
+** DONE test: io.py: cover
+** TODO ref: io.py: rename
+** TODO ref: PyRecordThread: write_on stuff
+** TODO depr: QueuedWriter: remove stop_record() and start_record()
+** TODO make: combine cover and check targets
+** TODO make: add todo target
+** TODO deploy: update coverage requirement
 * later
+** TODO feat: QueuedWriter: check write_on in init
 ** TODO other: add versions for the requirements
 ** TODO other: reqs: update pyqtgraph version
 ** TODO ref: widget testing: extract asker
